### PR TITLE
[doc] Refactoring doc of supported analyzers

### LINKED ISSUE #2555: Analyzer support documentation is a bit misleading
ON the main page the info about analyzers is a bit misleading:
https://github.com/Ericsson/codechecker/#code-analyzers-supported-by-codechecker

I propose the following table structure:

 | tool | analyer execution | storage of analyzer results | 
 | ------ | ---------------------- | ------------------------- | 
 | clangsa|yes|yes | 
 | cppcheck|no|yes | 

and please add a short explanation

CodeChecker can execute the analysis for the clang static analyzer and clang tidy analyzers only,  however it can store the the results of other analyzers too in the database. So it can be used as a generic tool for visualizing analyzer results.

### PR DESCRIPTION
**Pull request by csordasmarton**
> Closes #2555

Highligh in the documentation that CodeChecker can only execute ClangSA
and Clang Tidy analyzers but supports the storage of multiple analyzer
results.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -285,8 +292,9 @@ For now CodeChecker supports only `C/C++` related analyzers:
 |                | [Cppcheck](/tools/report-converter/README.md#cppcheck)    |
 |                | [Clang Sanitizers](supported_code_analyzers.md#clang-sanitizers)    |
 
-We are planning to support multiple analyzers of different programming
-languages. For more detailed information [see](supported_code_analyzers.md).
+We are planning to support the storage of multiple analyzers results of
```
Comment (gyorb) [inline docs/README.md]: Probably I would rephrase this sentence a little bit to something like this:
We are planning to support the storage of the results created by other analyzers for different languages (Java, Python ...).

File: docs/supported_code_analyzers.md [hunk 2]
```diff
@@ -3,11 +3,11 @@ CodeChecker can execute two main `C/C++` static analyzer tools:
 - [Clang Tidy](https://clang.llvm.org/extra/clang-tidy/)
 - [Clang Static Analyzer](https://clang-analyzer.llvm.org/)
 
-We are planning to support more analyzer tools by creating a separate converter
-tool which can be used to convert the output of different code analyzer tools
-to a CodeChecker result directory which can be stored to a CodeChecker server.
+We have created a separate [converter tool](/tools/report-converter) which
+can be used to convert the output of different code analyzer tools to a
```
Comment (gyorb) [inline docs/supported_code_analyzers.md]:
```suggestion
can be used to convert the output of different source code analyzer tools to a
```

File: docs/README.md [hunk 3]
```diff
@@ -285,8 +292,9 @@ For now CodeChecker supports only `C/C++` related analyzers:
 |                | [Cppcheck](/tools/report-converter/README.md#cppcheck)    |
 |                | [Clang Sanitizers](supported_code_analyzers.md#clang-sanitizers)    |
 
-We are planning to support multiple analyzers of different programming
-languages. For more detailed information [see](supported_code_analyzers.md).
+We are planning to support the storage of the results created by other
+analyzers for different languages (Java, Python ...). For more detailed
+information [see](supported_code_analyzers.md).
```
Comment (gyorb) [inline docs/README.md]: This sentence after rendering the markdown to html is a bit interesting: "For more detailed information [see](supported_code_analyzers.md)."
What to see is missing from the sentence.
Maybe some thing like this?
"For more detailed information check out the [supported code analyzers](supported_code_analyzers.md) documentation."

File: docs/README.md [hunk 4]
```diff
@@ -276,7 +276,14 @@ If you have Clang `3.7` installed you might see the following warning message:
    support bug identifier hash generation currently.
 
 # Code Analyzers supported by CodeChecker
-For now CodeChecker supports only `C/C++` related analyzers:
+CodeChecker can only execute two main `C/C++` static analyzer tools:
+- [Clang Tidy](https://clang.llvm.org/extra/clang-tidy/)
+- [Clang Static Analyzer](https://clang-analyzer.llvm.org/)
+
+However it can store the results of other analyzers tools in the database.
```
Comment (bruntib) [inline docs/README.md]:
```suggestion
However it can store the results of other analyzer tools in the database.
```
